### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by sb2nov**
@tpcstld PTAL

### CODE REVIEW

File: docs/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+=============
+
+`Dataduct <https://github.com/coursera/dataduct>`__ is a wrapper built
+on top of `AWS
+Datapipeline <http://docs.aws.amazon.com/datapipeline/latest/DeveloperGuide/what-is-datapipeline.html>`__
+which makes it easy to create ETL jobs. All jobs can be specified as a
+series of steps in a YAML file and would automatically be translated
+into datapipeline with appropriate pipeline objects.
+
+Top features include visualizing pipeline dependencies, extract data
```
Comment (tpcstld) [inline docs/introduction.rst]: The grammar in this paragraph needs to be changed.

extract -> extracting, Local -> local, run -> running, load -> loading, complete -> completing.

The final sentence is also not a sentence.

Comment (tpcstld) [inline docs/introduction.rst]: Alternatively, this should be made into a list :P

**Comment by tpcstld** [comment]
LGTM except for that one paragraph.

Man, the steps page has the best documentation ever, whoever wrote those docs is brilliant :P

**Comment by sb2nov** [comment]
That is true, you're like 100 times better at this than I am.

**Pull request merged**